modules/ROOT/pages/_partials/generic-local-mobile-dev-envir-reqs.adoc

```diff
@@ -41,7 +41,7 @@ You need the following installed on your machine:
 * [x] link:https://nodejs.org/[Node.js, window="_blank"] version 8.9.4 runtime and NPM
 * [x]  link:https://cordova.apache.org/docs/en/latest/guide/cli/#installing-the-cordova-cli[Cordova CLI, window="_blank"] version 8.0.0
 
-For information on how to set up your local development environment for Cordova, see link:https://cordova.apache.org/#getstarted[Cordova, window="_blank"].
+For information on how to set up a local Cordova development environment, see link:https://cordova.apache.org/#getstarted[Cordova, window="_blank"].
```

Comment:
see the link:https://cordova.apache.org/#getstarted[Cordova documentation, window="_blank"].